src/composition.jl

```diff
@@ -3,7 +3,7 @@ struct CompositeMap{T,As<:Tuple{Vararg{LinearMap}}} <: LinearMap{T}
     function CompositeMap{T,As}(maps::As) where {T,As}
         N = length(maps)
         for n = 2:N
-            size(maps[n],2)==size(maps[n-1],1) || throw(DimensionMismatch("CompositeMap"))
+            size(maps[n],2) == size(maps[n-1],1) || throw(DimensionMismatch("CompositeMap"))
         end
```

Comment:
👍

Reply:
Github interprets your thumb as a change request, so I'll pretend I addressed the change request by this stupid answer. :-D